crates/edr_generic/src/spec.rs

```diff
@@ -39,6 +43,72 @@ impl ChainSpec for GenericChainSpec {
     type SignedTransaction = crate::transaction::SignedWithFallbackToPostEip155;
 }
 
+impl GenericChainSpec {
+    fn blob_excess_gas_and_price(
```

Comment:
This function doesn't require any types or member functions from `GenericChainSpec`, so it can be a free method:

```suggestion
fn blob_excess_gas_and_price(
```